client/src/main.ts

```diff
@@ -145,6 +146,14 @@ async function setupApp(): Promise<void> {
       return [libparsec, nextStage];
     };
   } else if (import.meta.env.VITE_TESTBED_SERVER) {
```

Comment:
For me it's strange that the creation of testbed org are fixed on compilation:

If you set `TESTBED_SERVER` during build of the client, this `if` will always be true (`import.meta.env.FOO` is replace by it's value during build).
Meaning you need to recompile if you do not want this behavior